### PR TITLE
Content update - Read-only dependency properties (user story 1878471)

### PR DESCRIPTION
**Pull request by v-trisshores**
@adegeo, this PR is ready for your review.

Also updates the snippets for collection-type read-only dependency properties.

[Internal review link](https://review.docs.microsoft.com/en-us/dotnet/desktop/wpf/properties/read-only-dependency-properties?view=netdesktop-5.0&branch=pr-en-us-1230)

### CODE REVIEW

File: dotnet-desktop-guide/net/wpf/properties/snippets/read-only-dependency-properties/csharp/CodeSampleCsharp.csproj [hunk 1]
```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>WinExe</OutputType>
+    <TargetFramework>net5.0-windows</TargetFramework>
```
Comment (adegeo) [inline dotnet-desktop-guide/net/wpf/properties/snippets/read-only-dependency-properties/csharp/CodeSampleCsharp.csproj:5]: We should be using 6.0 now 😁

File: dotnet-desktop-guide/net/wpf/properties/snippets/read-only-dependency-properties/vb/CodeSampleVb.vbproj [hunk 2]
```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>WinExe</OutputType>
+    <TargetFramework>net5.0-windows</TargetFramework>
```
Comment (adegeo) [inline dotnet-desktop-guide/net/wpf/properties/snippets/read-only-dependency-properties/vb/CodeSampleVb.vbproj:5]: 6.0 here also

**Review by adegeo — CHANGES_REQUESTED** [review]
Lets change this to 6.0 in the source and then we're good to go. Great job!

File: dotnet-desktop-guide/net/wpf/properties/snippets/read-only-dependency-properties/csharp/CodeSampleCsharp.csproj (hunk 1 above)
Comment (v-trisshores) [reply dotnet-desktop-guide/net/wpf/properties/snippets/read-only-dependency-properties/csharp/CodeSampleCsharp.csproj]: Done.

File: dotnet-desktop-guide/net/wpf/properties/snippets/read-only-dependency-properties/vb/CodeSampleVb.vbproj (hunk 2 above)
Comment (v-trisshores) [reply dotnet-desktop-guide/net/wpf/properties/snippets/read-only-dependency-properties/vb/CodeSampleVb.vbproj]: Done.

**Comment by v-trisshores** [comment]
@adegeo, thank you for the review. :)

**Review by adegeo — APPROVED** [review]

**Pull request merged**